integrated_channels/blackboard/migrations/0013_alter_blackboardlearnerdatatransmissionaudit_index_together.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.2.15 on 2022-12-20 15:05
+
+from django.db import connection, migrations
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ('blackboard', '0012_move_and_recrete_completed_timestamp'),
+    ]
+
+    db_engine = connection.settings_dict['ENGINE']
+    if 'sqlite3' in db_engine:
+        operations = [
+            migrations.AlterIndexTogether(
+                name='blackboardlearnerdatatransmissionaudit',
+                index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
+            ),
+        ]
+    else:
+        operations = [
+            migrations.SeparateDatabaseAndState(
```

Comment:
for our stage/prod env - apply the normal migration to the state, but on the DB run our custom sql.